next.config.ts

```diff
@@ -0,0 +1,78 @@
+import type { NextConfig } from "next"
+
+const siteURL =
```

Comment:
I just recently moved siteURL to the library. you can delete this from here and update the env to remove `env.NEXT_PUBLIC_DEPLOY_URL`